launcher_scripts/conf/training/gpt3/126m.yaml

```diff
@@ -109,14 +109,15 @@ model:
   grad_allreduce_chunk_size_mb: 125
 
   ## Transformer Engine
+  # fp8 training is currently not supported in the improved models
```

Comment:
This is not an improved config. Let's leave it to True

Reply:
Done